_layouts/use-cases.html

```diff
@@ -9,6 +9,9 @@
             <div class="card-title display-6 mt-3 mb-4">Use Cases</div>
             <div class="row g-0">
                 <div class="col-auto">
+                  {% for use_case in site.use_cases %}
```

Comment:
There's probably a better way of achieving this :)

Reply:
@gracegrimwood do you know the right way to achieve this?  The other option would be to turn the use-case button into a drop down menu.

Reply:
What's the goal? To be able to navigate _to_ each use case, or to be able to navigate _within_ each use case? If you look at what's been done for the [docs layout](https://github.com/kroxylicious/kroxylicious.github.io/blob/main/_layouts/docs.html), with the internal navigation in a sidebar, does that sort of achieve what you're after?
If we want to repurpose that layout for the use cases we'll have to duplicate it, the docs layout is set up to process AsciiDoc rather than markdown.

Otherwise if you just want buttons that take you to each use case, there's always the [Slugify filter](https://jekyllrb.com/docs/liquid/filters/). Something like:
```
{% for use_case in site.use_cases %}
    {% include my_cool_button.html label=use_case.name %}
{% endfor %}
```
and then in `_includes/my_cool_button.html` you'd have something like this ([bootstrap button docs](https://getbootstrap.com/docs/5.3/components/buttons/)):
```
<button type="button" class="btn btn-primary" href="#{{ name | slugify }}">{{ name }}</button>
```
and then you just arrange those buttons however you'd like by adding structure around them with [grid components](https://getbootstrap.com/docs/5.3/layout/grid/) in `_layouts/use_cases.html`. The name property already exists in the existing use case front matter so it's just a matter of passing it through slugify to get something URL-safe.

Reply:
> What's the goal? To be able to navigate to each use case, or to be able to navigate within each use case? 

really the former.  currently, if you click on the use-case navigation you see "record encryption".   There's nothing obvious that  lets the reader know there's further use-cases below.  I vomited a TOC onto the page (sorry) as a quick and dirty solution.

Unfortunately, I don't have the jekyll skills (or the available bandwidth right now to learn).. so if you'd be able to make a suggestion, that would be appreciated.